### PR TITLE
Fix signup and login

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,6 +38,6 @@ group :development, :test do
   gem 'launchy'
   gem 'pry'
   gem 'database_cleaner'
-   gem 'better_errors'
-   gem 'binding_of_caller'
+  gem 'better_errors'
+  gem 'binding_of_caller'
```
Comment (kpearson) [inline Gemfile]: We like better errors

File: Gemfile.lock [hunk 2]
```diff
@@ -133,6 +134,11 @@ GEM
       cocaine (~> 0.5.3)
       mime-types
     pg (0.18.1)
+    poltergeist (1.5.1)
```
Comment (kpearson) [inline Gemfile.lock]: Lets test that js

File: Gemfile [hunk 3]
```diff
@@ -29,6 +29,7 @@ gem 'momentjs-rails', '>= 2.9.0'
 gem 'bootstrap3-datetimepicker-rails', '~> 4.0.0'
 
 group :development, :test do
+  gem 'poltergeist'
```
Comment (kpearson) [inline Gemfile]: Test that js

**Comment by bryceholcomb** [comment]
This is going to make the authentication spec run really slow because Poltergeist is testing the javascript, I would advise just running specific test files until you absolutely need to run all the tests haha

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -21,8 +21,6 @@
 //= require_tree .
 
 
-$(function(){ $(document).foundation(); });
-
```
Comment (kpearson) [inline app/assets/javascripts/application.js]: Bye bye foundation

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  <%= f.text_field :about_me %>
+
+  <%= f.text_field :email %>
+
+  <%= f.password_field :password %>
+
+  <%= f.password_field :password_confirmation %>
+
+  <%= f.submit "Sign Up" %>
+<% end %>
```
Comment (kpearson) [inline app/views/users/new.html.erb]: Are moving user new to its own view?

File: db/seeds.rb [hunk 6]
```diff
@@ -10,7 +10,6 @@
                about_me: "valid",
                display_name: "Alice",
                password: "password",
-               id: 1,
                slug: "Alice"},
```
Comment (kpearson) [inline db/seeds.rb]: we dont need to set no ids/

File: spec/feature/authentication_spec.rb [hunk 7]
```diff
@@ -27,9 +29,12 @@
       fill_in "user[first_name]", with: "Joe"
       fill_in "user[last_name]", with: "Doe"
       fill_in "user[email]", with: "Joe@example.com"
+      fill_in "user[display_name]", with: "anothername"
+      fill_in "user[about_me]", with: "desc"
```
Comment (kpearson) [inline spec/feature/authentication_spec.rb]: This is good I want to know more about people

**Pull request merged**